docs/index/index.rst

```diff
@@ -17,6 +17,14 @@ Daml Documentation
    :hidden:
    :caption: Home
 
+.. toctree::
+   :titlesonly:
+   :maxdepth: 0
+   :hidden:
+   :caption: Daml Finance
+
+   Daml Finance <https://digital-asset.github.io/daml-finance/>
+
```

Comment:
We probably still want a small overview page that includes a link rather than linking directly from the toc.

Reply:
I was hoping for a top level link that didn't need a dropdown (couldn't get that working) and I wasn't even sure where to put this in the TOC. 

I'll add a small overview page for this.

Reply:
Added